pkg/chart/loader/archive.go

```diff
@@ -233,3 +241,18 @@ func LoadArchive(in io.Reader) (*chart.Chart, error) {
 
 	return LoadFiles(files)
 }
+
+// ParseEnvSettings reads environment variables and updates the size limits
+func parseEnvSettings() {
+	if chartSizeStr := os.Getenv("HELM_MAX_DECOMPRESSED_CHART_SIZE"); chartSizeStr != "" {
```

Comment:
I'm not sure if switching to cmd flag is a good idea. I see those settings as something exceptionnal and a global setup of helm commands.
Also loader package is pretty exempt from any cmd flags and any configuration values. What do you think ?

Reply:
Maybe outside the scope of concern, but I believe argo uses the helm executable to template charts:
https://github.com/argoproj/argo-cd/blob/cde5b4b7d40683d84df98994e67c4cbd9b247255/util/helm/helm.go#L69-L75

With a cli arg, it should be possible to set this on a per-chart basis if needed, I think.  With an env var, it's kind of all or nothing unless argo enhances its API to expose env vars to the helm process when templating